recipes-mmx/klish/klish_2.1.4.bb

```diff
@@ -0,0 +1,86 @@
+# (except for failure to satisfy the conditions of this license) patent license
+# to make, have made, use, offer to sell, sell, import, and otherwise transfer
+# this software, where such license applies only to those patent claims, already
+# acquired or hereafter acquired, licensable by such copyright holder or contributor
+# that are necessarily infringed by:
+#
+# (a) their Contribution(s) (the licensed copyrights of copyright holders and
+# non-copyrightable additions of contributors, in source or binary form) alone;
+# or
+#
+# (b) combination of their Contribution(s) with the work of authorship to which
+# such Contribution(s) was added by such copyright holder or contributor, if,
+# at the time the Contribution is added, such addition causes such combination
+# to be necessarily infringed. The patent license shall not apply to any other
+# combinations which include the Contribution.
+#
+# Except as expressly stated above, no rights or licenses from any copyright
+# holder or contributor is granted under this license, whether expressly, by
+# implication, estoppel or otherwise.
+#
+# DISCLAIMER
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+# USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# NOTE
+#
+# This is part of a management middleware software package called MMX that was developed by Inango Systems Ltd.
+#
+# This version of MMX provides web and command-line management interfaces.
+#
+# Please contact us at Inango at support@inango-systems.com if you would like to hear more about
+# - other management packages, such as SNMP, TR-069 or Netconf
+# - how we can extend the data model to support all parts of your system
+# - professional sub-contract and customization services
+#
+################################################################################
+
+DESCRIPTION = ""
+HOMEPAGE = "http://klish.libcode.org"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://LICENCE;md5=763c2d89173e7008fab9b7ecf2605ab5"
+SECTION = "utils"
+DEPENDS = "xz-native expat"
+
+SRC_URI = "http://libcode.org/attachments/download/70/;downloadfilename=klish-${PV}.tar.xz"
+SRC_URI[sha256sum] = "a89dd1027dce713407b6d68e836c8fdead56406dcfc650da84da8e0b92c9b2e5"
+
+inherit autotools
+
+S = "${WORKDIR}/klish-${PV}"
+
+FILES_${PN} += "/usr/lib"
```

Comment:
Does not it get to FILES_${PN} by default as declared in `meta/conf/bitbake.conf` ?

Reply:
This is case when Yocto doesn't add so file from that folder because it's a plugin so library
If remove this line then bitbake will print next error message:
```
WARNING: klish-2.1.4-r0 do_package: QA Issue: klish: Files/directories were installed but not shipped in any package:
  /usr/lib/clish_plugin_clish.so
```

I'll add full path  to that so file into `FILES_${PN}`

Reply:
fixed